.github/scripts/build-cuda.sh

```diff
@@ -4,9 +4,13 @@ declare build_os
 declare cuda_version
 
 set -xeuo pipefail
-build_capability="50;52;60;61;70;75;80;86;89;90"
-[[ "${cuda_version}" == 11.7.* ]] && build_capability=${build_capability%??????}
-[[ "${cuda_version}" == 11.8.* ]] && build_capability=${build_capability%???}
+build_capability="50;52;60;61;70;75;80;86;89;90;100;120"
+remove_for_11_7=";86;89;90;100;120"
```

Comment:
```suggestion
remove_for_11_7=";89;90;100;120"
```

We can keep sm86 here.